merico-dev/lake
# feat: Adding make, grafana and docker to our project

# Summary

This brings over some things we need from the old project.
- Make (we used to use package.json but that does not work in Go)
- Grafana (This is just the config of the dashboards)
- Devops folder (just a docker-compose file)
- Github templates (issues and pull requests)
- Github Workflow for CZ (`make commit`)
- .gitignore

### Key Points


### Description
We need a way to run complex commands easily. Enter `make`.

You can run these commands with things like `make compose`

We also need our `.github` directory and `/grafana`

We also need our docker-compose. You can run this with `make compose`

### How to test this

1. Run `make compose`
2. See all services started in docker
